swerve_point_generator/Coefs[] orient_coefs
swerve_point_generator/Coefs[] x_coefs
swerve_point_generator/Coefs[] y_coefs
int32[] spline_groups
float64[] wait_before_group
float64[] t_shift
bool[] flip
bool[] x_invert
float64[] end_points
float64 initial_v
float64 final_v
---
trajectory_msgs/JointTrajectory joint_trajectory
bool[] hold
float64 dt
